pip/__init__.py

```diff
@@ -4,6 +4,7 @@
 import logging
 import os
 import optparse
+import pkg_resources
```

Comment:
This is not used and removing it will fix the pep8 builds and one of the failing tests.